File: julia/04-control-flow/conditionals.jl
# if / elseif / else
temperature = 18

if temperature > 30
    println("It's hot")
elseif temperature > 15
    println("It's mild")
else
    println("It's cold")
end

# if is an expression: it returns the value of the last evaluated branch
status = if temperature > 0
    "above freezing"
else
    "freezing or below"
end
println("Water is $status")

# Conditions must be Bool — Julia does NOT treat 0 or "" as false
if isodd(7)
    println("7 is odd")
end

File: r/04-control-flow/conditionals.R
# Conditionals in R
temperature <- 18

if (temperature > 25) {
  print("It's hot")
} else if (temperature >= 15) {
  print("It's mild")
} else {
  print("It's cold")
}

# In R, if is an expression - it returns a value you can assign
status <- if (temperature >= 15) "comfortable" else "chilly"
print(paste("Status:", status))
